graylog2-web-interface/src/components/alertconditions/AlertConditionForm.jsx

```diff
@@ -37,8 +37,9 @@ const AlertConditionForm = React.createClass({
     return (
       <Well className="alert-type-form alert-type-form-message-count form-inline well-sm">
         Title: <input ref="title" type="text" className="form-control" autoComplete="off" defaultValue={alertCondition.title}/>
-        {' '}
-        <small>This title can also be included in alert notifications.</small>
+        <span style={{ color: '#939393', marginLeft: 10 }}>
```

Comment:
I only do that sometimes when the styling is only used in one place. Anyway, you are right, I'll put it into some file 👍